src/components/layout/organize/SingleCampaignLayout.tsx

```diff
@@ -0,0 +1,32 @@
+import { FunctionComponent } from 'react';
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+
+import getCampaign from '../../../fetching/getCampaign';
+import TabbedLayout from './TabbedLayout';
+
+interface SingleCampaignLayoutProps {
+    fixedHeight?: boolean;
+}
+
+const SingleCampaignLayout: FunctionComponent<SingleCampaignLayoutProps> = ({ children, fixedHeight }) => {
+    const { campId, orgId } = useRouter().query;
+    const campaignQuery = useQuery(['campaign', orgId, campId ], getCampaign(orgId  as string, campId as string));
+
+    return (
+        <TabbedLayout
+            baseHref={ `/organize/${orgId}/campaigns/${campId}` }
+            fixedHeight={ fixedHeight }
+            subtitle="subtitle for single campaign tabs" //TODO: generate subtitle
```

Comment:
Can this be easily done using the existing code for the date range on the summary page? Otherwise lets clear the string, but keep the `TODO` comment.